### ISSUE TITLE
Filtering corpus for correct and incorrect sentences

### ISSUE THREAD

**Issue by baarsrj** [languagetool-org/languagetool#1215]
If would be great if one could pass a corpus through LT to 
- divide it into sentences just like LT would
- separate sentences with hits from those without (2 output files)

By checking the file with no errors detected, and detecting remaining by hand or algorithms, more rules could be found.

I know this can also be done (partly) by making a program and using the API. But it would be more convenient if it were simply in a LT command or option.

**Comment by dpelle** [comment]
I disagree. LT job's is to find mistake in input text. It should only do that, to keep it simple and avoid feature creep. What you describe should in my opinion rather be done by scripting outside of LT for example.

**Comment by baarsrj** [comment]
The checking functionality is available form the command line; but also the tagging functionality is. The SRX sentence splitting functionality is not available from the command line, and thus hard to test and improve because of that. So I guess we agree to disagree.
